pkg/specgen/generate/pod_create.go

```diff
@@ -195,6 +195,9 @@ func MapSpec(p *specgen.PodSpecGenerator) (*specgen.SpecGenerator, error) {
 			p.InfraContainerSpec.NetworkOptions = p.NetworkOptions
 			p.InfraContainerSpec.NetNS.NSMode = specgen.Slirp
 		}
+	case specgen.Path:
+		logrus.Debugf("Pod will use namespace path networking")
+		p.InfraContainerSpec.NetNS.NSMode = specgen.Path
```

Comment:
I think you also need to populate `p.InfraContainerSpec.NetNS.Value` as well, which holds the path of the namespace.

Reply:
Do you mean copy it from the pod spec:
```
p.InfraContainerSpec.NetNS.Value = p.PodNetworkConfig.NetNS.Value
```
`MapSpec()` does not have a test yet, do you think I should add one?

Reply:
RE Copying - yes.

RE Test - I don't think it's strictly necessary, but I would definitely not complain if you added one

Reply:
`NetNS.Value` is set and I added a test for `MapSpec`.